webapp/src/js/reducers/index.js

```diff
@@ -40,16 +37,17 @@
         return Object.assign({}, state, {
           selected: merge({}, state.selected, action.payload.selected)
         });
-      case actionTypes.UPDATE_SELECTED_ITEM:
-        selected = state.selected.map(item => {
+      case actionTypes.UPDATE_SELECTED_ITEM: {
```

Comment:
I'd never seen this before (curly braces in `case` statements), makes sense for variable scoping!